aspnetcore/blazor/components/render-modes.md

```diff
@@ -555,27 +477,37 @@ app.MapRazorComponents<App>()
 
 The `@rendermode` directive takes a single parameter that's a static instance of type <xref:Microsoft.AspNetCore.Components.IComponentRenderMode>. The `@rendermode` directive attribute can take any render mode instance, static or not. The Blazor framework provides the <xref:Microsoft.AspNetCore.Components.Web.RenderMode> static class with some predefined render modes for convenience, but you can create your own.
 
-Consider the following example that creates a shorthand Interactive Server render mode without prerendering:
+Normally, a component uses the following `@attribute` directive to [disable prerendering](#prerendering):
+
+```razor
+@attribute [RenderModeInteractiveServer(prerender: false)]
+```
+
+However, consider the following example that creates a shorthand Interactive Server render mode without prerendering via the app's `_Imports` file (`Components/_Imports.razor`):
 
 ```csharp
 public static IComponentRenderMode InteractiveServerWithoutPrerendering { get; } = 
     new InteractiveServerRenderMode(prerender: false);
 ```
 
-Use the shorthand render mode in components:
+Use the shorthand render mode in components throughout the `Components` folder:
 
 ```razor
 @rendermode InteractiveServerWithoutPrerendering
 ```
 
-> [!NOTE]
-> Normally, a component uses the following `@attribute` directive to disable prerendering:
->
-> ```razor
-> @attribute [RenderModeInteractiveServer(prerender: false)]
-> ```
->
-> For more information, see the [Prerendering](#prerendering) section.
+Alternatively, a single component instance can define a custom render mode via a private field:
```

Comment:
@SteveSandersonMS, @MackinnonBuck what will be the behavior if after rendering one component, the value of the static field changes - will the next instance of the component to be rendered use the updated render mode?